bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -396,7 +396,13 @@ private Switch createPlayerButtons() {
                         }
                     }
 
-                    if (state instanceof QuantityType) {
+                    if (state instanceof DecimalType) {
+                        // for DecimalTypes we don't want to risk format exceptions, if pattern contains unit
+                        // placeholder
+                        if (formatPattern.contains(UnitUtils.UNIT_PLACEHOLDER)) {
+                            formatPattern = formatPattern.replaceAll(UnitUtils.UNIT_PLACEHOLDER, "");
```

Comment:
Should we maybe also trim it, so that the pattern does not have a trailing space?

Reply:
Yes, I thought about that too. Will add it.